File: convertToTitle.py
class Solution:
# Approach 1- Math 
    def convertToTitle(self, columnNumber): # TC O(log n) SC O(1)
        ans=""
        while columnNumber>0:
            columnNumber-=1
            ans+=  chr((columnNumber%26 + ord('A')))
            columnNumber//=26
        
        return ans[::-1]

s=Solution()
print(s.convertToTitle(902))

File: convertToTitle.cpp
#include <iostream>
#include <string>
using namespace std;

// Approach 1- Math
class Solution { // TC O(log n) SC O(1)
public:
    string convertToTitle(int columnNumber) {
        string ans;
        while(columnNumber>0){
            columnNumber--;
            ans += (char) (columnNumber%26 + 'A');
            columnNumber/=26;
        }
        reverse(ans.begin(), ans.end());
        return ans;
    }
};

int main(){
    Solution s;
    cout << s.convertToTitle(902) << endl;
    return 0;
}